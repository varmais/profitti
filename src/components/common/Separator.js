import React from 'react';
import { PixelRatio, StyleSheet, View } from 'react-native';
import config from '../../config';

const styles = StyleSheet.create({
  separator: {
    height: 1 / PixelRatio.get(),
    backgroundColor: config.colors.graySubtle
  }
});

export default () => (
  <View style={styles.separator} />
);
