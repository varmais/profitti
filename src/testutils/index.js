import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

global.React = React;
global.renderer = renderer;

export function createNavigationProp (params = {}) {
  return {
    state: {params: params},
    navigate: () => {}
  }
}