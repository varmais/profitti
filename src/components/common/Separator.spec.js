import '../../testutils';
import Separator from './Separator';

describe('components:common:Separator', () => {
  test('renders Separator', () => {
    const snapshot = renderer.create(<Separator />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});