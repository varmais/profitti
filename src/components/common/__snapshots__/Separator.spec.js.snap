// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:common:Separator renders Separator 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#e6e6e6",
      "height": 0.5,
    }
  }
/>
`;
